<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemStock</class>
 <widget class="QMainWindow" name="ItemStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ItemStock</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxLogin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>361</width>
      <height>154</height>
     </rect>
    </property>
    <property name="title">
     <string>Log in to change Item in database</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelItemCode">
         <property name="text">
          <string>Item Code</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditItemCode"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelItemCode_2">
         <property name="text">
          <string>Password  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPassword"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>117</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonLogin">
       <property name="text">
        <string>Log in</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>116</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
